log/gps/gps_destinator_misc.ksy

```diff
@@ -0,0 +1,95 @@
+meta:
+  id: gps_destinator
+  title: Destinator mixed stuff format
+  application: Destinator
+  file-extension:
+    - rte
+    - dat
+  license: GPL-2.0-or-later
+  encoding: utf-16le
+  endian: le
+
+doc: |
+  Destinator formats
```

Comment:
This is very short, especially with the format title just saying "mixed stuff format" 🙂 A summary of what exactly this "mixed stuff" is would be helpful, and perhaps a short explanation of what Destinator is exactly.